Index: src/pages/Navigation.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/Navigation.tsx b/src/pages/Navigation.tsx
--- a/src/pages/Navigation.tsx	(date 1618508383990)
+++ b/src/pages/Navigation.tsx	(date 1618508383990)
@@ -19,6 +19,11 @@
   const { params } = match;
   const { page } = params;
 
+  const tabNameToIndex = {
+    0: 'regulations',
+    1: 'publications',
+  };
+
   const getIndexToTabName = {
     regulations: 0,
     publications: 1,
@@ -31,12 +36,9 @@
   );
 
   const handleChange = (event: any, newValue: number) => {
-    if (selectedTab === 0) {
-      history.push('/regulations');
-    }
-    if (selectedTab === 1) {
-      history.push('/publications');
-    }
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    history.push(`/${tabNameToIndex[newValue]}`);
     setSelectedTab(newValue);
   };
 
